auditbeat/buildkite.yml

```diff
@@ -0,0 +1,50 @@
+when:
+    changeset:                 ## when PR contains any of those entries in the changeset
+        - "^auditbeat/.*"
+        - "@ci"                ## special token regarding the changeset for the ci
+        - "@oss"               ## special token regarding the changeset for the oss
+stages:
+    # mandatory stage - it runs always for:
+    #   - branches/tags
+    #   - on PRs
+    #   - GitHub comment /test auditbeat
+    #   - GitHub label auditbeat
+    mandatory:
+        # NOTE: stage name should be unique!
+        unitTest:
+            command: "mage build unitTest"
+            platform: "family/core-ubuntu-2204"
+        crosscompile:
```

Comment:
This step can be forced to run using:

* The GitHub comment `/test auditbeat crosscompile`
* The GitHub label `auditbeat-crosscompile`